### PR TITLE
Update readme.md to show how to generate files

### PR DESCRIPTION
**Pull request by ptmkenny**
Based on the comment by @skizzo here: https://github.com/itgalaxy/favicons/issues/159#issuecomment-471275884

This really should be documented because many people like me may try to use this without being familiar with Node.js, and such info can save us a lot of trouble.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
 
 For the full list of files, check `config/files.json`. For the full HTML code, check `config/html.js`. Finally, for the full list of icons, check `config/icons.json`.
 
+To save the files that are output, you can use `fs`.
+
+For example, amend the code above for Node.js as follows:
+
+```
+var fs = require ("fs");
```
Comment (andy128k) [inline readme.md:178]: `const` is preferable over `var`.

File: readme.md [hunk 2]
```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
...
+
+try {
+  const response = await favicons(source, configuration);
+  response.images.map (image => {
+    fs.writeFileSync (`./your-favicons-directory/${image.name}`, image.contents)
```
Comment (andy128k) [inline readme.md:183]: Here is an inconsistency with an invocation of `favicons` line where `await` is used.

File: readme.md [hunk 3]
```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
...
+To save the files that are output, you can use `fs`.
+
+For example, amend the code above for Node.js as follows:
+
+```
+var fs = require ("fs");
+
+try {
+  const response = await favicons(source, configuration);
+  response.images.map (image => {
```
Comment (andy128k) [inline readme.md:182]: Example is not full. There could be also `files` and `html` fragments.

File: readme.md (hunk 1 above)
Comment (andy128k) [reply readme.md:178]: While we support CommonJS modules, it's better to promote ECMAScript modules.

File: readme.md [hunk 4]
```diff
@@ -170,6 +170,24 @@ import { config } from "favicons";
...
+    fs.writeFileSync (`./your-favicons-directory/${image.name}`, image.contents)
+  })
+
+} catch (error) {
```
Comment (andy128k) [inline readme.md:186]: That may be not needed in an example. There is nothing specific to `favicons` in the `catch` block.

**Review by andy128k — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (ptmkenny) [reply readme.md]: I updated the PR to use ECMAScript.

File: readme.md (hunk 3 above)
Comment (ptmkenny) [reply readme.md]: I'm not sure how you want to handle this.  Feel free to change the code as you see fit.

Comment (ptmkenny) [reply readme.md]: @Elysiome Thanks! I will close this PR in favor of yours.

**Pull request closed without merging**